File: app/src/main/java/com/neome/java/api/meta/base/dto/DefnDtoChartXAxis.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdChartXAxis;
import com.neome.java.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnDtoChartXAxis
{
  @Nullable
  public DefnDtoColor color;

  @Nullable
  public MetaIdField colorFieldId;

  @Nullable
  public DefnDtoColor colorVar;

  @Nullable
  public String label;

  public MetaIdChartXAxis metaId;

  @Nullable
  public String valueOptionId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnDtoChartXAxis.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnDtoColor
import com.neome.api.meta.base.Types.MetaIdChartXAxis
import com.neome.api.meta.base.Types.MetaIdField

interface DefnDtoChartXAxis
{
  val color: DefnDtoColor?
  val colorFieldId: MetaIdField?
  val colorVar: DefnDtoColor?
  val label: String?
  val metaId: MetaIdChartXAxis
  val valueOptionId: String?
}